doc/source/data/api/dataset.rst

```diff
@@ -147,9 +147,8 @@ Schema
 .. autosummary::
    :nosignatures:
    :toctree: doc/
-   :template: autosummary/class_without_autosummary_noindex.rst
 
-   Schema
+   ~dataset.Schema
```

Comment:
@can-anyscale how does this change affect what we're trying to do with templates, if at all?

Reply:
actually good catch @bveeramani , the doc shows `ray.data.dataset.Schema` as the name of the API (https://anyscale-ray--46279.com.readthedocs.build/en/46279/data/api/doc/ray.data.dataset.Schema.html#ray.data.dataset.Schema). Maybe just drop the `~dataset`, the api should just be `ray.data.Schema`

Reply:
Ahh, sorry about this - I thought you and @bveeramani were concerned with how the link appeared on the API reference page:

![image](https://github.com/ray-project/ray/assets/14017872/de13aaaa-c1ae-41da-b1ec-e6b215c01024)

but it seems like it also affects the Schema doc page itself as well. After testing, it seems like just passing `Schema` to autosummary works just fine. Thanks for catching this!